### DISCUSSION TITLE
Incremental LOF feature

### DISCUSSION THREAD

**Discussion by KatFT** [online-ml/river#1436] [General]
Hello!

I'm doing a thesis regarding the theme of anomaly detection in the context of data stream. I've discovered from river amazing algorithms that I can explore to detects anomalies but I've noticed that you have an algorithm that is not included in the API documentation and that is the incremental LOF algorithm. I want to know if this algorithm is already functional to test or if is not ready yet.
Your response will be very useful!
Regards!

**Comment by smastelini** [comment]
Hi @KatFT. LOF will oficially debut in the next release. Meanwhile, you can install the development version of river to access this functionality. 

You can also check the [dev docs](https://riverml.xyz/dev/api/anomaly/LocalOutlierFactor/).

**Reply by MaxHalford** [reply]
I will release v0.20 this weekend :)
